File: PortfolioApp/Models/Main_models/About.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace PortfolioApp.Models
{
    public class About
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [StringLength(maximumLength: 50, ErrorMessage = "Максимальна довжина введеної стрічки - 50 символів")]
        [MaxLength(50)]
        public string Title { get; set; }
        public string ImageName { get; set; }

        [Required]
        public string Description { get; set; }

        [Required]
        public string Profile_desc { get; set; }

        [Required]
        [StringLength(maximumLength: 50, ErrorMessage = "Максимальна довжина введеної стрічки - 50 символів")]
        [MaxLength(50)]
        public string FullName { get; set; }

        [Required]
        public DateTime Birth_date { get; set; }

        [Required]
        public string Job { get; set; }

        [Required]
        public string Website { get; set; }

        [Required]
        public string Email { get; set; }

        [Required]
        public string Skills_desc { get; set; }
    }
}

File: PortfolioApp/ClientApp/src/app/Models/Main_models/About.ts
import { DatePipe } from '@angular/common';

export class About {
  constructor(
    public id?: number,
    public title?: string,
    public imageName?: string,
    public description?: string,
    public profile_desc?: string,
    public fullname?: string,
    public birth_date?: DatePipe,
    public job?: string,
    public website?: string,
    public email?: string,
    public skills_desc?: string) { }
}
